src/composables/sdk13.ts

```diff
@@ -164,6 +153,31 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     };
   }
 
+  watch(
+    activeAccount,
+    (oldVal, newVal) => {
+      if (!isEqual(oldVal, newVal) && sdk) {
+        sdk._pushAccountsToApps();
+      }
+    },
+    { deep: true },
+  );
```

Comment:
I don't think it's a good pattern that we watch the accounts and perform actions as this composable is not always loaded, so the watcher can miss some updates.

Reply:
Regarding our discussion I think we can replace it with some kind of `onActiveAccountChange` hook returned by the accounts composable.

Reply:
We have parked this optimization for better research.